README.md

```diff
@@ -1,6 +1,63 @@
 # [ng-suggestion](http://ianwalter.github.io/ng-suggestion/)
 *Flexible AngularJS typeahead / autocomplete / suggestion / predictive search directive*
 
-I don't have time for docs right now just read the source.
-
+A wise man once said, *"I don't have time for docs right now just read the source."*  
 &mdash;&nbsp; [Ian Kennington Walter](http://iankwalter.com)
+
+#### Dependencies
+* [ng-dropdown](https://github.com/flashnotes/ng-dropdown/)
+
+#### Installing ng-suggestion
+
+Install using Bower:  
+```
+bower install ng-suggestion --save
+```
+
+Include js/ng-suggestion.min.js in your app. You may also want to include
+css/ng-suggestion.min.css, but it's not necessary, only recommended.
+
+#### Setup for ng-suggestion
+
+```javascript
+var app = angular.module('your-wonderful-app', ['ng-suggestion'])
```

Comment:
Let's follow Angular's style conventions for naming modules. See for example those defined here: https://docs.angularjs.org/guide/module.